### PR TITLE
fix(config): #578 Onboard Silva to FAM

### PR DESCRIPTION
**Pull request by ConradBoydElliottGustafson**

### CODE REVIEW

File: server/flyway/sql/V28__create_silva_application.sql [hunk 1]
```diff
@@ -0,0 +1,58 @@
+-- Create silva_dev, silva_test and silva_prod applications
+INSERT INTO app_fam.fam_application (
+    application_name,
+    application_description,
+    app_environment,
+    create_user,
+    create_date
+)
+VALUES ('SILVA_DEV', ' (DEV)', 'DEV', CURRENT_USER, CURRENT_DATE),
```
Comment (ianliuwk1019) [inline server/flyway/sql/V28__create_silva_application.sql:9]: Do you want to change their app to "RESULT" not "SILVA"?  I am just affraid silva will have more than one applications to be onboard to FAM in future and "**RESULT**" is the application name but "SILVA" is their team name.

**Review by ianliuwk1019 — CHANGES_REQUESTED** [review]

File: server/flyway/sql/V28__create_silva_application.sql (hunk 1 above)
Comment (gormless87) [reply server/flyway/sql/V28__create_silva_application.sql:9]: Silva is Team Silva's chosen name for 'New Results'

Comment (ianliuwk1019) [reply server/flyway/sql/V28__create_silva_application.sql:9]: Ah~! Sorry Conrad. I didn't know that. Thanks @gormless87 !

Comment (ianliuwk1019) [inline server/flyway/sql/V28__create_silva_application.sql:9]: " (DEV)" Missing application_description for 'DEV' environment.
And the TEST/PROD description looks like SPAR (`Seed Planning and Registry Application`)

**Review by ianliuwk1019 — CHANGES_REQUESTED** [review]

**Comment by ianliuwk1019** [comment]
Hi @ConradBoydElliottGustafson , I assume we will still need to do an update for this task later since I see "infrastructure/server/oidc_clients_silva.tf" file for production, you use "localhost" so we don't know yet their production callback url.

File: server/flyway/sql/V28__create_silva_application.sql (hunk 1 above)
Comment (ConradBoydElliottGustafson) [reply server/flyway/sql/V28__create_silva_application.sql]: Yep, copy paste error

**Comment by ConradBoydElliottGustafson** [comment]
> Hi @ConradBoydElliottGustafson , I assume we will still need to do an update for this task later since I see "infrastructure/server/oidc_clients_silva.tf" file for production, you use "localhost" so we don't know yet their production callback url.

Yes. for all their clients, they will give us the actual values down the road.

**Review by ianliuwk1019 — APPROVED** [review]

**Pull request merged**